json-path/src/main/java/com/jayway/jsonpath/Predicate.java

```diff
@@ -23,6 +24,8 @@
 
     boolean apply(PredicateContext ctx);
 
+    boolean check(TokenStack stack, int idx);
```

Comment:
I think he means that the interface should only have 1 method, which is what allows the use with lambdas in the Java 8 onwards. eg so you cannot add a new check method.